pkg/oc/cli/admin/migrate/storage/storage.go

```diff
@@ -248,6 +249,26 @@ func (o *MigrateAPIStorageOptions) Complete(f kcmdutil.Factory, c *cobra.Command
 		}
 	}
 
+	clientConfig, err := f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	// We do not have a way to access the REST client that dynamic.NewForConfig uses
+	// Thus we cannot use resource.NewClientWithOptions with our flowcontrol.NewFakeAlwaysRateLimiter
+	// To avoid any possibility of rate limiting, use an absurdly high burst and QPS
+	// We handle throttling via our own limiter
+	clientConfigCopy := rest.CopyConfig(clientConfig)
+	clientConfigCopy.Burst = 1 << 24
```

Comment:
really?  How about a real number?  All 9s!

Reply:
`2^24` is a real number :stuck_out_tongue: